src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlSecurityUtility.cs

```diff
@@ -316,14 +311,14 @@ internal static int GetBase64LengthFromByteLength(int byteLength)
         /// <summary>
         /// Verifies Column Master Key Signature.
         /// </summary>
-        internal static void VerifyColumnMasterKeySignature(string keyStoreName, string keyPath, string serverName, bool isEnclaveEnabled, byte[] CMKSignature)
+        internal static void VerifyColumnMasterKeySignature(string keyStoreName, string keyPath, string serverName, bool isEnclaveEnabled, byte[] CMKSignature, SqlConnection connection)
```

Comment:
Since SqlConnection is now as a parameter, there is no need to have additional serverName here. It can be obtained from SqlConnection directly.